1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,10 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents =
+  group.filter((name) => !students.includes(name)).length == 0;
+
```

Comment:
This is good, but look at the readme for the exercise, it was expecting use of the `array.every()` method